src/components/Contributors.tsx

```diff
@@ -31,52 +31,31 @@ const Contributors = () => {
         have contributed so far!
       </p>
 
-      <Flex flexWrap="wrap">
+      <Flex className="flex-wrap">
         {contributorsList.map((contributor) => (
           <LinkBox
-            key={contributor.login}
             as="div"
-            maxWidth="132px"
-            margin="2"
-            boxShadow="0px 14px 66px rgba(0, 0, 0, 0.07), 0px 10px 17px rgba(0, 0, 0, 0.03), 0px 4px 7px rgba(0, 0, 0, 0.05)"
-            _hover={{
-              textDecoration: "none",
-              borderRadius: "base",
-              boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
-              background: "tableBackgroundHover",
-              transition: "transform 0.1s",
-              transform: "scale(1.02)",
-            }}
-            _focus={{
-              textDecoration: "none",
-              borderRadius: "base",
-              boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
-              background: "tableBackgroundHover",
-              transition: "transform 0.1s",
-              transform: "scale(1.02)",
-            }}
+            className="m-2 max-w-[132px] transform shadow-table transition-transform duration-100 hover:scale-[1.02] hover:rounded hover:bg-background-table-hover hover:no-underline hover:shadow-table-box-hover focus:scale-[1.02] focus:rounded focus:no-underline focus:shadow-table-box-hover"
```

Comment:
We can deprecate `tableBackgroundHover` and use `bg-background-highlight` for this. Would help on cleaning these styles a bit.

We can also remove the shadow effects and the underline wouldn't be necessary since we are overriding that on the link itself.

```suggestion
            className="m-2 max-w-[132px] transform shadow transition-transform duration-100 hover:scale-[1.02] hover:rounded hover:bg-background-highlight focus:scale-[1.02] focus:rounded"
```

Reply:
This is noted